src/ol_infrastructure/components/services/vault.py

```diff
@@ -134,3 +137,81 @@ def __init__(
                 default_ttl=db_config.default_ttl
             )
         self.register_outputs({})
+
+class OLVaultAWSSecretsEngineConfig(BaseModel):
+    app_name: Text
+    access_key: Text
+    default_lease_ttl_seconds: int = SIX_MONTHS
+    max_lease_ttl_seconds: int = SIX_MONTHS
+    description: Text
+    secret_key: Text
+    path: Text
```

Comment:
I think it would be good to make the `path` attribute more descriptive to indicate what type of path it is supposed to be. e.g. `mount_path` or `vault_backend_path`, etc.

Reply:
I'm partially of the mindset that if you're creating a secrets engine mount and you're not aware of what `path` is then you probably should spend some time reading the vault docs prior to using this. Additionally, feel like it's easier to debug when the values we're using map to the same ones in pulumi and hopefully in vault.